pages/docs/reference/data-classes.md

```diff
@@ -66,6 +66,33 @@ val jack = User(name = "Jack", age = 1)
 val olderJack = jack.copy(age = 2)
 ```
 
+## Excluding Properties
```

Comment:
The title and the section placement do not look consistent to me. 

The first section also mentions this behavior as "The compiler automatically derives the following members from all properties declared in the primary constructor". 

Probably this should be titled as "Properties declared in the class body" and placed the second.